index.js

```diff
@@ -0,0 +1,31 @@
+var path = require("path");
+var fs = require("fs");
+var prettyPrint = require("html").prettyPrint;
+var chai = require("chai");
+chai.config.includeStack = true;
+var expect = chai.expect;
```

Comment:
Ah just spotted all this should be removed from this file, as we're not in a test context.